src/components/pages/CaseView/CaseActivity.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+
+export default function CaseActivity() {
+  // When using real data, please comment out the dummyData, and initialize the caseHistoryData to the history props passed in.
+
+  const historyDummyData = [
+    { action: 'opened', date: '12-03-2009' },
+    { action: 'changed', date: '11-07-2009' },
+    { action: 'changed', date: '03-23-2009' },
+    { action: 'closed', date: '08-16-2019' },
+  ];
+
+  // State is used to determine whether the Expanded View should be rendered on the DOM
+  const [expandView, setExpandView] = useState(false);
+  const [caseHistoryData, setCaseHistoryData] = useState(historyDummyData);
+
+  const toggleExpandedView = () => {
+    setExpandView(!expandView);
+  };
+
+  return (
+    <>
+      <div>
+        <button style={{ marginLeft: '1em' }} onClick={toggleExpandedView}>
+          View Case Activity
+        </button>
+      </div>
+      {expandView && (
+        <ExpandedActivity
+          caseHistoryData={caseHistoryData}
+          toggleExpandedView={toggleExpandedView}
+        />
+      )}
+    </>
+  );
+}
+
+// A custom styles object is used to not clog or interfere with the css files used site-wide, and allowing localized styling changes.
+const expandedStyles = {
+  div: {
```

Comment:
I suggest to move out the css from here to style.less where all component css is being added by convention to the project.

Reply:
Awesome Thanks! Done!